libinitializer/CMakeLists.txt

```diff
@@ -21,5 +21,5 @@ endif()
 add_library(initializer ${SRC_LIST} ${HEADERS})
 target_include_directories(initializer SYSTEM BEFORE PUBLIC ${BOOST_INCLUDE_DIR})
 
-target_link_libraries(initializer PUBLIC channelserver rpc ledger TBB devcore)
+target_link_libraries(initializer PUBLIC channelserver rpc ledger TBB devcore eventfilter)
```

Comment:
In my opinion, the ledger should depend on eventfilter, so that initializer will depend on ledger instead of eventfilter

Reply:
the ledger has no need to depend on eventfilter.